lua/core/utils/init.lua

```diff
@@ -270,20 +272,29 @@ end
 -- @param module the module path of the override setting
 -- @param default the default settings that will be overridden
 -- @param extend boolean value to either extend the default settings or overwrite them with the user settings entirely (default: true)
--- @param prefix a module prefix for where to search (default: user)
+-- @param prefixes a module prefixes for where to search (default: { "user" })
 -- @return the new configuration settings with the user overrides applied
-function astronvim.user_plugin_opts(module, default, extend, prefix)
+function astronvim.user_plugin_opts(module, default, extend, prefixes)
   -- default to extend = true
   if extend == nil then extend = true end
   -- if no default table is provided set it to an empty table
   default = default or {}
-  -- try to load a module file if it exists
-  local user_settings = load_module_file((prefix or "user") .. "." .. module)
-  -- if no user module file is found, try to load an override from the user settings table from user/init.lua
-  if user_settings == nil and prefix == nil then user_settings = user_setting_table(module) end
-  -- if a user override was found call the configuration engine
-  if user_settings ~= nil then default = func_or_extend(user_settings, default, extend) end
-  -- return the final configuration table with any overrides applied
+
+  -- if nil, then default to { "user" }
+  prefixes = prefixes or { "user" }
```

Comment:
oh right, there needs to be a way to actually override this easily. in my testing, I just override it directly in the file, which obviously is ready for merge. any suggestions?

Reply:
@mehalter

Reply:
Yeah this needs to be configurable without modifying the core files. This is completely unsupported with our updater and is not going to be supported in the future. We need to have clear separation between the core files and user files

Reply:
A shell variable is the best I can think of. not much else